### PR TITLE
Feature/condo/sberdoma 113/property add edit view

### PR DESCRIPTION
**Pull request by dkoviazin**
1. Create property
2. View property
3. Edit property
createProperty
viewProperty
editProperty
chessboard

### CODE REVIEW

File: apps/condo/domains/property/components/BasePropertyForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    return (
+        <>
+            <FormWithAction 
+                action={action} 
+                initialValues={initialValues} 
+                validateTrigger={['onBlur', 'onSubmit']}
+                formValuesToMutationDataPreprocessor={ formData => {
+                    try {
+                        // address is created or changed
+                        const addressMeta = { dv: 1, ...JSON.parse(formData['address']) }
```
Comment (Dimitreee) [inline apps/condo/domains/property/components/BasePropertyForm/index.tsx:41]: Add try/catch to JSON.parse

File: apps/condo/domains/property/components/panels/BuilderPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { useIntl } from '@core/next/intl'
```
Comment (Dimitreee) [inline apps/condo/domains/property/components/panels/BuilderPanel.tsx:1]: What is `panels`?
Add readme with description in components/panels/

File: apps/condo/domains/property/components/panels/ResidentPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import { Col, Row, Typography, Space, Tooltip } from 'antd'
+import React from 'react'
+import { Button } from '@condo/domains/common/components/Button'
+
+const EmptyResidentBlock: React.FC = () => {
+    const intl = useIntl()
+    const EmptyResidentsTitle = intl.formatMessage({ id: 'pages.condo.property.form.EmptyResidents.header' })
+    const EmptyResidentsMessage = intl.formatMessage({ id: 'pages.condo.property.form.EmptyResidents.text' })
+    const UploadFile = intl.formatMessage({ id: 'UploadFile' })
+    const NotImplementedYedMessage = intl.formatMessage({ id: 'NotImplementedYed' })
```
Comment (Dimitreee) [inline apps/condo/domains/property/components/panels/ResidentPanel.tsx:12]: `NotImplementedYed`, `Yed`...
This is my mistake, can you, please, rename all `Yed` occurrences for `Yet`?)

File: apps/condo/pages/property/[id]/index.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+const PropertyIdPage: IPageWithHeaderAction = () => {
+    const intl = useIntl()
+    const PageTitleMsg = intl.formatMessage({ id: 'pages.condo.property.id.PageTitle' })
+    const ServerErrorMsg = intl.formatMessage({ id: 'ServerError' })
+
+    const router = useRouter()
+    let { query: { id } } = router
+    if (typeof id !== 'string') {
```
Comment (Dimitreee) [inline apps/condo/pages/property/[id]/index.tsx:47]: can u, please, add same check to `apps/condo/pages/ticket/[id]/index.tsx:149` and remove type cast?

File: apps/condo/domains/property/schema/Property.js [hunk 5]
```diff
@@ -102,6 +102,25 @@ const Property = new GQLListSchema('Property', {
                 return count
             },
         },
+
+        ticketsClosed: {
```
Comment (Dimitreee) [inline apps/condo/domains/property/schema/Property.js:106]: add tests

File: apps/condo/pages/property/[id]/update.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+            </Head>
+            <PageWrapper>
+                <PageContent>
+                    <OrganizationRequired>
+                        <Row gutter={[12, 40]}>
+                            <Col span={24}>
+                                <Typography.Title level={1} style={{ margin: 0 }}>{PageTitleMsg}</Typography.Title>
+                            </Col>
+                            <Col span={24}>
+                                <PropertyForm id={id as string}/>
```
Comment (Dimitreee) [inline apps/condo/pages/property/[id]/update.tsx:34]: Add same check as at apps/condo/pages/property/[id]/index.tsx:47, remove type cast

File: apps/condo/domains/property/components/BasePropertyForm/index.tsx (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/property/components/BasePropertyForm/index.tsx:41]: Done

File: apps/condo/domains/property/components/panels/BuilderPanel.tsx (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/domains/property/components/panels/BuilderPanel.tsx:1]: Done

File: apps/condo/domains/property/components/panels/ResidentPanel.tsx (hunk 3 above)
Comment (dkoviazin) [reply apps/condo/domains/property/components/panels/ResidentPanel.tsx:12]: Done

File: apps/condo/pages/property/[id]/index.tsx (hunk 4 above)
Comment (dkoviazin) [reply apps/condo/pages/property/[id]/index.tsx:47]: I don't know how to fix it without wrighting lot's of typescript code redeclaring all starting from NextRouter. If you have some suggestions - they are welcome
```js
const { query: { id } } = useRouter()
const { loading, obj: property, error } = useObject({ where: { id: id as string } })
```

File: apps/condo/pages/property/[id]/update.tsx (hunk 6 above)
Comment (dkoviazin) [reply apps/condo/pages/property/[id]/update.tsx:34]: Same here - all suggestions are welcome
```
const { query: { id } } = useRouter() // can be string or string[]
```

File: apps/condo/pages/property/[id]/index.tsx (hunk 4 above)
Comment (Dimitreee) [reply apps/condo/pages/property/[id]/index.tsx:47]: `if (typeof id !== 'string') {
    id = String(id)
  }` 
   Its ok

File: apps/condo/domains/property/schema/Property.js (hunk 5 above)
Comment (dkoviazin) [reply apps/condo/domains/property/schema/Property.js:106]: Done

**Review by Dimitreee — APPROVED** [review]

**Pull request merged**